### DISCUSSION TITLE
Does vercel support DNSSEC?

### DISCUSSION THREAD

**Discussion by exrobbie** [vercel/vercel#8660] [Help]
Hi I've deployed many APPs on vercel and want to use the DNSSEC function on cloudflare, but it says "the cname redirect to cname.vercel-dns.com don't have DNSSEC enabled", could you instruct me how I can get it set on vercel? And is it safe if I don't enable DNSSEC because vercel has already done this for me? Thank you.

**Comment by amyegan** [comment]
Hi @earthbeyond! Sorry you are running into this problem. I'm not able to replicate it, but I'm probably just not following the right steps to see what you are seeing. Can you share an outline of the steps you took that led to this error? That will help me better understand what is going wrong.

**Comment by emilianobonassi** [comment]
running https://dnsviz.net/ shows currently vercel-dns.com not supporting dnssec
